contrib/nydusify/cmd/nydusify.go

```diff
@@ -364,7 +364,18 @@ func main() {
 					Usage:   "Read prefetch list from STDIN, please input absolute paths line by line",
 					EnvVars: []string{"PREFETCH_PATTERNS"},
 				},
-
+				&cli.StringFlag{
+					Name:    "compressor",
+					Value:   "zstd",
+					Usage:   "Algorithm to compress image data blob, possible values: none, lz4_block, zstd",
+					EnvVars: []string{"COMPRESSOR"},
+				},
+				&cli.StringFlag{
+					Name:    "chunk-size",
+					Value:   "",
```

Comment:
Maybe give a default value here, so that users can know in the help message.

Reply:
fixed